content/200-concepts/100-components/02-prisma-client/190-deployment.mdx

```diff
@@ -135,10 +135,17 @@ Prisma Client depends on the [query engine](query-engine) that is running as a b
 
 The query engine is implemented in Rust and is used by Prisma in the form of executable binary files. The binary is downloaded when `prisma generate` is called. If the call happens on a different environment when deploying your Prisma-based application to production, you need to ensure that the `binaryTarget` in the Prisma schema is compatible with your production environment as described in [binary targets](/reference/api-reference/prisma-schema-reference#binarytargets-options).
 
-<!-- A configuration with binaryTargets = ["native", "linux-glibc-libssl1.0.2"] is only needed when the development machine is also the machine responsible to build for production but the platform in production is different, like AWS lambda, now, etc. -->
 
-<!-- | Cloud Platform | Binary target        |
-| :------------- | :------------------- |
-| AWS Lambda     | `rhel-openssl-1.0.x` |
-| Netlify        | `rhel-openssl-1.0.x` |
-| Zeit Now       | `rhel-openssl-1.0.x` | -->
+## AWS Elastic Beanstalk
+
+AWS Elastic Beanstalk is a PaaS-like deployment service that abstracts away infrastructure and allows you to deploy applications to AWS quickly.
+
+When deploying an app using Prisma Client to Beanstalk, Prisma generates the Prisma Client code into `node_modules`. This is typically done in the `postinstall` hook.
+
+Because Beanstalk limits the ability to write to the filesystem in the `postinstall` hook, you need to create an [`.npmrc`](https://docs.npmjs.com/cli/v6/configuring-npm/npmrc) file in the root of your project and add the following configuration:
+
+```config file=.npmrc
+unsafe-perm=true
+```
+
+Enabling `unsafe-perm` forces _npm_ to as _root_, avoiding the filesystem access problem, thereby allowing the `prisma generate` command in the `postinstall` hook to generate code into `node_modules`.
```

Comment:
`Enabling `unsafe-perm` forces _npm_ to as _root_`
I think something needs to be changed here (looking for an idea)